warehouse/templates/manage/account/publishing.html

```diff
@@ -188,6 +188,77 @@
   </form>
 {% endmacro %}
 
+{% macro activestate_form(request, pending_activestate_pubisher_form) %}
+  {{ form_error_anchor(pending_activestate_pubisher_form) }}
+  <form method="POST" action="{{ request.route_path('manage.account.publishing') }}#errors">
+    <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+    {{ form_errors(pending_activestate_pubisher_form) }}
+    <div class="form-group">
+      <label for="project_name" class="form-group__label">
+        {% trans %}PyPI Project Name{% endtrans %}
+        {% if pending_activestate_pubisher_form.project_name.flags.required %}
+        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+        {% endif %}
+      </label>
+      {{ pending_activestate_pubisher_form.project_name(placeholder=gettext("project name"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="project_name-errors") }}
+      <p class="form-group__help-text">
+        {% trans %}The project (on PyPI) that will be created when this publisher is used{% endtrans %}
+      </p>
+      <div id="project_name-errors">
+        {{ field_errors(pending_activestate_pubisher_form.project_name) }}
+      </div>
+    </div>
+    <div class="form-group">
+      <label for="organization" class="form-group__label">
+        {% trans %}Organization{% endtrans %}
+        {% if pending_activestate_pubisher_form.organization.flags.required %}
+        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+        {% endif %}
+      </label>
+      {{ pending_activestate_pubisher_form.organization(placeholder=gettext("my-organization"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="organization-errors") }}
+      <p class="form-group__help-text">
+        {% trans %}The ActiveState organization name that owns the project{% endtrans %}
+      </p>
+      <div id="organization-errors">
+        {{ field_errors(pending_activestate_pubisher_form.organization) }}
+      </div>
+    </div>
+    <div class="form-group">
+      <label for="project" class="form-group__label">
+        {% trans %}ActiveState Project name{% endtrans %}
+        {% if pending_activestate_pubisher_form.project.flags.required %}
+        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+        {% endif %}
+      </label>
+      {{ pending_activestate_pubisher_form.project(placeholder=gettext("my-project"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", **{"aria-describedby":"project-errors"}) }}
```

Comment:
nit: move `aria-describedby` out of a splat at to a proper keyword `aria_describedby`

Similar note applies to the rest of the templates.

Reply:
oh wow....good eye!